### ISSUE TITLE
install fails on several Linux systems

### ISSUE THREAD

**Issue by jorol** [LibreCat/Catmandu-Fix-Date#2]
Install fails on several Linux systems, see http://matrix.cpantesters.org/?dist=Catmandu-Fix-Date+0.0124

Error message:
```terminal
t/00-load.t .................. ok
Invalid locale code or name: en-EN
t/01-datetime_format.t ....... 
Dubious, test returned 255 (wstat 65280, 0xff00)
```

**Comment by nicolasfranck** [comment]
Thanks for reporting this. I'll look into it.
Seems like some linux distributions do not have
enough localization information.

**Comment by jorol** [comment]
Is 'en_EN' a valid code or should it be something like 'en_GB' or 'en_US'?

```terminal 
$ locale -a
...
en_AG
en_AU
en_AU.utf8
en_BW
en_BW.utf8
en_CA
en_CA.utf8
en_DK
en_DK.iso885915
en_DK.utf8
en_GB
en_GB.iso885915
en_GB.utf8
en_HK
en_HK.utf8
en_IE
en_IE@euro
en_IE.utf8
en_IN
en_NG
en_NZ
en_NZ.utf8
en_PH
en_PH.utf8
en_SG
en_SG.utf8
en_US
en_US.iso885915
en_US.utf8
en_ZA
en_ZA.utf8
en_ZM
en_ZW
en_ZW.utf8
...
```

**Comment by nicolasfranck** [comment]
You're right. I don't see it listed on my system either.
No idea where I got this from. EN isn't a country code ;-)
Something deprecated that used to work in the past?
Anyway I see reports on the internet about the same mistake.

The default for datetime is now UTC, so make en_GB the default locale?

**Comment by jorol** [comment]
OK

**Comment by nics** [comment]
isn't en_US a more sensible choice?

**Comment by nicolasfranck** [comment]
Better?

http://matrix.cpantesters.org/?dist=Catmandu-Fix-Date+0.0126

**Comment by jorol** [comment]
Perfect! Thanks.
